README.md

```diff
@@ -348,7 +348,45 @@ meetingSession.audioVideo.realtimeSubscribeToVolumeIndicator(
 );
 ```
 
-**Use case 11.** Detect the most active speaker. For example, you can enlarge the active speaker's video element if available.
+**Use case 11.** Handle `realtimeSubscribeToVolumeIndicator` updates efficiently.
+
+When subscribed to `realtimeSubscribeToVolumeIndicator`, the values are updated if they are changed, else they will be null. To efficiently implement this API, you can listen separately to each of these values. Consequently, your callback will only handle the value, you are interested in.
+
+- To track just mute:
+```js
+const presentAttendeeId = meetingSession.configuration.credentials.attendeeId;
```

Comment:
Add this comment from use case 10. You can subscribe to any other attendee's volume change.

```
// This is your attendee ID. You can also subscribe to another attendee's ID.
// See the "Attendees" section for an example on how to retrieve other attendee IDs
// in a session.
const presentAttendeeId = meetingSession.configuration.credentials.attendeeId;
```

Reply:
Cool, thanks I have added this in the code block.